UPGRADE-3.x.md

```diff
@@ -4,6 +4,10 @@ UPGRADE 3.x
 UPGRADE FROM 3.x to 3.x
 =======================
 
+### Deprecations
+
+- `Symfony\Component\Translation\TranslatorInterface` is deprecated in favor of `Symfony\Contracts\Translation\TranslatorInterface`
```

Comment:
A good upgrade message should contain the reference where passing this class is deprecated.